fdbserver/workloads/SpecialKeySpaceCorrectness.actor.cpp

```diff
@@ -70,14 +70,12 @@ struct SpecialKeySpaceCorrectnessWorkload : TestWorkload {
 	double getCheckTimeout() const override { return std::numeric_limits<double>::max(); }
 
 	void disableFailureInjectionWorkloads(std::set<std::string>& out) const override {
-		out.insert("RandomMoveKeys");
-
-		// Rollback interferes with the
-		// \xff\xff/worker_interfaces test, since it can
-		// trigger a cluster recvoery, causing the worker
-		// interface for a machine to be updated in the middle
-		// of the test.
-		out.insert("RollbackWorkload");
+		// Failure injection workloads like Rollback, Attrition and so on are interfering with the test.
+		// In particular, the test aims to test special keys' functions on monitoring and managing the cluster.
+		// It expects the FDB cluster is healthy and not doing unexpected configuration changes.
```

Comment:
Is the requirement for a healthy cluster a limitation of the test, or of the special key space feature? When somebody uses the special key space, they don't necessarily know whether the cluster is healthy, and so we should return reasonable answers in that case as well.

Reply:
yeah, that's right. Users don't know whether the cluster is healthy.
This test is written for correctness verification, like if the result of the work interfaces is consistent or if the coordinators' change is expected as we give.
Any recovery/reboot changes that happened in the background are unknown to this test,
so we cannot guarantee the results' consistency if that happens.

I think it's better to have a robustness test for special key features to test their results when the cluster is unhealthy or under recovery. 
But that should be a separate workload, this one was written for very basic correctness verification like is read results consistent and in order.

Reply:
I discussed with @sfc-gh-anoyes for this,
we think it makes sense to have a second workload to test the APIs under the failures like you suggested.
Let's merge this first, as it may block others' CI.
I have a second PR to add a workload to test the robustness of the special keys API